book/src/user_guide/html_report.md

```diff
@@ -1,5 +1,7 @@
 # HTML Report
 
+### You should use feature "html_reports" for this.
+
```

Comment:
I agree with @theCollectiv, especially for users who have not worked with Cargo features much yet it would be good to explicitly show how to enable this. Also accessibility-wise this probably should not be a header (`###` prefix). What about the following?
````suggestion
Starting with Criterion.rs 0.4.0 HTML reports must be explicitly enabled via the `html_reports` [feature](https://doc.rust-lang.org/cargo/reference/features.html#dependency-features):
```toml
[dev-dependencies]
criterion = {version = "0.4.0", features = ["html_reports"] }
```
````

Reply:
Having this PR landed would've saved me a ton of time because it wasn't clear to me in the docs why I wasn't seeing html_reports. https://stackoverflow.com/questions/75808243/rust-criterion-wont-create-report-html